rpm/wpa_supplicant-Revert-Mark-authorization-completed-on-driver-indica.patch

```diff
@@ -0,0 +1,50 @@
+From 2514856652f9a393e505d542cb8f039f8bac10f5 Mon Sep 17 00:00:00 2001
+From: Janne Grunau <janne-fdr@jannau.net>
+Date: Sun, 4 Aug 2024 13:24:42 +0200
+Subject: [PATCH 1/1] Revert "Mark authorization completed on driver indication
```

Comment:
Added in fedora wpa_supplicant git

---
commit a5b0db1716b7b4e63a297dcb5b10e98317c54205 (HEAD -> rawhide, origin/rawhide, origin/main, origin/f41, origin/HEAD)
Author: Janne Grunau <janne-fdr@jannau.net>
Date:   Sun Aug 4 14:54:24 2024 +0200

    Revert commit breaking authentication on brcmfmac
    
    Resolves: #2302577
    
    Signed-off-by: Janne Grunau <janne-fdr@jannau.net>
---

Googling brcmfmac it seems to be a PCI device thing. Not applicable for us?

Reply:
Ok, so it is not a generic Broadcom issue. Do we even have any devices that use Broadcom chipset/WLAN? If not, we could consider to drop this.

Reply:
Does the patch break anything on other devices? If not I'd consider keeping the patch in case there are devices which would have Broadcom chipsets, .i.e community porter devices.

Reply:
Doesn't look like a broadcom hardware issue but a specific driver issue: 
https://lore.kernel.org/asahi/20240803-brcmfmac_pmksa_del_ssid-v1-1-4e85f19135e1@jannau.net/T/#u

Reply:
Well, https://lists.infradead.org/pipermail/hostap/2024-August/042941.html it does affect other devices too. There is some offloading that is handled differently by different drivers. 

>That means the patched Infineon/Cypress driver is broken as
well ... It might be only brcmfmac,

So a lot of speculation. We could search if any of the kernel trees have `NL80211_EXT_FEATURE_SAE_OFFLOAD ` and/or `NL80211_EXT_FEATURE_SAE_OFFLOAD_AP`.

And the last message was from Aug. 10.

Reply:
Dropped after testing with S19ProMaxS. It seemed to work better without this.